### PR TITLE
fix: allow text-selecting in dialogs & reset cursor

### PR DESCRIPTION
**Pull request by Rafi993**
## What does it do ?

1. Enable text selection in keyboard modal
2. Convert cursor inside modal from crosshair to text

### CODE REVIEW

**Comment by lipis** [comment]
Do we really want that.. it affects all dialogs.

**Comment by dwelle** [comment]
Which is good, no? I suspect the fact we disallow text selection is a regression, not a feature.

File: src/components/Dialog.scss [hunk 1]
```diff
@@ -1,6 +1,11 @@
 @import "../css/_variables";
 
 .excalidraw {
+  .Dialog {
+    user-select: text;
+    cursor: text;
```
Comment (lipis) [inline src/components/Dialog.scss:6]: This affects the whole dialog.. can we remove the cursor?

**Comment by Rafi993** [comment]
> Do we really want that.. it affects all dialogs.

Right now the cursor and text selection is disabled in the body tag affecting the whole app.

File: src/components/Dialog.scss (hunk 1 above)
Comment (Rafi993) [reply src/components/Dialog.scss:6]: but I will show cross hair cursor when user hovers over the modal

**Comment by lipis** [comment]
Here is what I mean... the cursor shouldn't apply everywhere (if at all)

Kapture 2021-01-14 at 15 11 54

File: src/components/Dialog.scss (hunk 1 above)
Comment (Rafi993) [reply src/components/Dialog.scss:6]: I think I could add a new class for keyboard dialog and add that cursor there,

Comment (Rafi993) [reply src/components/Dialog.scss:6]: That might be better alternative. Any thoughts ?

**Comment by Rafi993** [comment]
> Here is what I mean... the cursor shouldn't apply everywhere (if at all)
> 
> Kapture 2021-01-14 at 15 11 54

@lipis I have removed ```cursor:text``` now

**Review by lipis — APPROVED** [review]

File: src/components/Dialog.scss [hunk 2]
```diff
@@ -1,6 +1,10 @@
 @import "../css/_variables";
 
 .excalidraw {
+  .Dialog {
+    user-select: text;
```
Comment (dwelle) [inline src/components/Dialog.scss:5]:
```suggestion
    user-select: text;
    cursor: auto;
```

Comment (dwelle) [reply src/components/Dialog.scss:5]: good catch on the `text` cursor. Should be set to `auto` instead.

Comment (Rafi993) [reply src/components/Dialog.scss:5]: Thank you @dwelle. I have done the change now. should I squash the commits before merge?

Comment (lipis) [reply src/components/Dialog.scss:5]: no.. we squash before merging

File: src/components/Dialog.scss [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+    cursor: auto;
```
Comment (lipis) [inline src/components/Dialog.scss:6]: isn't that the default one?

Comment (dwelle) [reply src/components/Dialog.scss:6]: not if we override it from upstream (when changing cursor on scene)

**Comment by lipis** [comment]
Merge?

**Pull request merged**